sig-windows/charter.md

```diff
@@ -14,13 +14,14 @@ Windows specific implementation.
 
 #### Code, Binaries and Services
 
-- Windows specific code in all parts of the codebase.
+- Windows specific code in all parts of the codebase
 - Testing of Windows specific features and clusters
 
 #### Cross-cutting and Externally Facing Processes
 
 - Work with other SIGs on areas where Windows and Linux (and possibly other OSes in the future) deviate from one another in terms of functionality.
-
+- Providing community guidance for external tools and paradigms necessary for production Windows k8s administration
+- Providing a reference implementation(s) of an end-to-end integrated Windows cluster that can be used for R&D, prototyping, and testing by other SIGs or companies interested in building k8s Windows integrations (this is motivated by the fact that container runtime technologies may change over time and the community benefits from having a canonical example of how to use Windows with K8s).
```

Comment:
i would remove that last sentence...IMHO we are not committing to a reference implementation.